frontend-cert/js-projects/dates-lab/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should create a variable named `currentDate` using `let` and assign it the current date and time using the `Date` object.
+1. You should create a function named `formatDateMMDDYYYY` that takes a date object as a parameter and returns the date formatted as `MM/DD/YYYY`.
+1. Inside the `formatDateMMDDYYYY` function, you should create variables named `month`, `day`, and `year` using `let`.
+1. The `month` variable should be assigned a String value of the month from the date object using the `getMonth` method and adding 1 to the result, and a `padStart` method to ensure the month is always two digits.
+1. The `day` variable should be assigned a String value of the day from the date object using the `getDate` method and a `padStart` method to ensure the day is always two digits.
+1. The `year` variable should be the date object's full year using the `getFullYear` method.
+1. You should return a template literal string that uses the `month`, `day`, and `year` variables to format the date as `MM/DD/YYYY`.
+1. You should create a function named `formatDateLong` that takes a date object as a parameter.
+1. Inside the `formatDateLong` function, you should create a variable named `options` using `let`.
+1. Assign `options` an object with a `year` property set to `numeric`, a `month` property set to `long`, and a `day` property set to `numeric`.
+1. You should return a string that uses the `toLocaleDateString` method on the date object with the `options` object as an argument.
```

Comment:
The `toLocaleDateString` won't be taught.
So this user story either needs to be changed

Reply:
If you feel like this method should be taught, then we can add it.
But you will have to let me know 👍

Reply:
Okay, will update the user stories rather

Reply:
> If you feel like this method should be taught, then we can add it. But you will have to let me know 👍

I think we can teach the method, getting rid of it will mean I have to rewrite the function in a longer format, but am open to other suggestions too

Reply:
the lecture video script has been updated to mention toLocaleDateString